data/migrations/20210802234515_add-episodes-table.js

```diff
@@ -0,0 +1,17 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('Episodes', (t) => {
+    t.increments('ID');
+    t.integer('StoryID')
+      .notNullable()
+      .unsigned()
+      .references('StoriesNew.ID')
+      .onDelete('CASCADE');
+    t.integer('EpisodeNumber');
```

Comment:
Let's create a new migration to alter the name of this field. Currently to access the field you have to type `episodes.episodenumber` putting the table name in the field is redundant. For basic queries, we just want `episodes.number`. Advanced queries with joins can rename the field in the select clause.

Reply:
That's a good point, we were going for clarity when looking at the db itself but your comment makes sense. 👍🏻